Container image from FIPS docs doesn't work

The example container image from the FIPS docs https://www.keycloak.org/server/fips#_keycloak_server_in_fips_mode_in_the_container does not work. Build of such container on RHEL 9.1 with the command like:
```
podman build . -t my-fips-keycloak
```

fail with:
```
[1/2] STEP 6/7: RUN cp /tmp/files/kc.java.security /opt/keycloak/conf/
--> 212af4b6b94
[1/2] STEP 7/7: RUN /opt/keycloak/bin/kc.sh build --features=fips --fips-mode=strict
The DelayedHandler was closed before any children handlers were configured. Messages will be written to stderr.
2023-02-23 13:58:21,593 DEBUG [org.jboss.logging] (main) Logging Provider: org.jboss.logging.JBossLogManagerProvider

Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:61)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:32)
Caused by: java.lang.ExceptionInInitializerError
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.lang.Class.getEnumConstantsShared(Class.java:3837)
	at java.base/java.lang.Class.getEnumConstants(Class.java:3812)
	at org.keycloak.config.OptionBuilder$2.<init>(OptionBuilder.java:92)
	at org.keycloak.config.OptionBuilder.expectedValues(OptionBuilder.java:91)
	at org.keycloak.config.OptionBuilder.<init>(OptionBuilder.java:51)
	at org.keycloak.config.FeatureOptions.<clinit>(FeatureOptions.java:12)
	at org.keycloak.quarkus.runtime.configuration.mappers.FeaturePropertyMappers.getMappers(FeaturePropertyMappers.java:25)
	at org.keycloak.quarkus.runtime.configuration.mappers.PropertyMappers.<clinit>(PropertyMappers.java:37)
	at org.keycloak.quarkus.runtime.cli.Picocli.addOptionsToCli(Picocli.java:401)
	at org.keycloak.quarkus.runtime.cli.Picocli.addCommandOptions(Picocli.java:377)
	at org.keycloak.quarkus.runtime.cli.Picocli.createCommandLine(Picocli.java:346)
	at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:85)
	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:88)
	... 6 more
Caused by: java.security.ProviderException: Could not initialize NSS
	at jdk.crypto.cryptoki/sun.security.pkcs11.SunPKCS11.<init>(SunPKCS11.java:284)
	at jdk.crypto.cryptoki/sun.security.pkcs11.SunPKCS11$1.run(SunPKCS11.java:179)
	at jdk.crypto.cryptoki/sun.security.pkcs11.SunPKCS11$1.run(SunPKCS11.java:153)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at jdk.crypto.cryptoki/sun.security.pkcs11.SunPKCS11.configure(SunPKCS11.java:153)
	at java.base/sun.security.jca.ProviderConfig$3.run(ProviderConfig.java:257)
	at java.base/sun.security.jca.ProviderConfig$3.run(ProviderConfig.java:248)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at java.base/sun.security.jca.ProviderConfig.doLoadProvider(ProviderConfig.java:248)
	at java.base/sun.security.jca.ProviderConfig.getProvider(ProviderConfig.java:226)
	at java.base/sun.security.jca.ProviderList.loadAll(ProviderList.java:317)
	at java.base/sun.security.jca.ProviderList.removeInvalid(ProviderList.java:334)
	at java.base/sun.security.jca.Providers.getFullProviderList(Providers.java:186)
	at java.base/java.security.Security.getProviders(Security.java:537)
	at java.security.jgss/sun.security.jgss.ProviderList.<init>(ProviderList.java:128)
	at java.security.jgss/sun.security.jgss.GSSManagerImpl.<init>(GSSManagerImpl.java:56)
	at java.security.jgss/org.ietf.jgss.GSSManager.getInstance(GSSManager.java:152)
	at org.keycloak.common.util.KerberosJdkProvider.isKerberosAvailable(KerberosJdkProvider.java:68)
	at org.keycloak.common.Profile$Feature.<clinit>(Profile.java:84)
	... 23 more
Caused by: java.io.IOException: NSS initialization failed
	at jdk.crypto.cryptoki/sun.security.pkcs11.Secmod.initialize(Secmod.java:243)
	at jdk.crypto.cryptoki/sun.security.pkcs11.SunPKCS11.<init>(SunPKCS11.java:279)
	... 41 more
Error: error building at STEP "RUN /opt/keycloak/bin/kc.sh build --features=fips --fips-mode=strict": error while running runtime: exit status 1
```

It might be a regression of https://github.com/keycloak/keycloak/pull/16879 or something else...

--- a/quarkus/container/ubi-null.sh
+++ b/quarkus/container/ubi-null.sh
@@ -31,7 +31,6 @@ python3-libs
 python3-pip-wheel
 python3-setuptools-wheel
 p11-kit
-sqlite-libs
 
 EOF
 

